.gitignore

```diff
@@ -10,6 +10,7 @@ _built
 *.swo
 *.css.map
 *.js.map
+report.*.json
```

Comment:
I'd have a slight preference for adding this to your own personal `.gitignore` file (often located at `~/.gitignore`), but I guess this is okay too.

Reply:
I think we might have added it to both during a call with Sondra. My bad.